pkg/cli/importimage/importimage.go

```diff
@@ -39,7 +39,17 @@ var (
 		see the 'tag' command.`)
 
 	importImageExample = templates.Examples(`
+		# Update imported data for an already existing image stream.
 		oc import-image mystream
+
+		# Import image tags into a new image stream.
+		oc import-image mystream --from=registry.io/repo/image --all --confirm
+
+		# Import image tags into a new image stream using a custom timeout.
+		oc --request-timeout=5m import-image mystream --from=registry.io/repo/image --all --confirm
+
+		# Import image tag into a new image stream.
+		oc import-image mystream:latest --from=registry.io/repo/image:latest --confirm
```

Comment:
It'll work only if mystream:latest doesn't exist, right?

Reply:
No, it works even if "mystream:latest" exists.